### DISCUSSION TITLE
Black-Out Internal Macbok screen while keeping sensor adjustments on external screens

### DISCUSSION THREAD

**Discussion by DavidMali** [MonitorControl/MonitorControl#966] [General]
Hey there,

thanks a ton for your app, it's been a blessing.
The main feature I use it for is the ability to adjust the external monitor brightness according to the light sensor in my MacBook.
However, there is one situation that doesn't really work right now:

Often I want to watch a movie on my external display and turn the brightness on my MacBook screen all the way down using MonitorControl to black it out. This has the desired effect but also stops the sensor from making adjustments to the external monitor. Is it at all possible to blackout the internal MacBook screen while keeping the sensor functionality on the external one?
Maybe something with a completely black overlay on the internal screen to trick the system into thinking a brightness is set and therefore still adjusting it using the sensor?

Thanks a lot for your app and your time!

**Comment by waydabber** [comment]
Hi @M0mentoMemes - something like this could be added technically - this would result in the macbook backlight still be activated however at a lower level.

**Reply by the0neyouseek** [reply]
@M0mentoMemes To add on this, as this is quite a specific feature, I'm not sure we'll do it. In the meantime, I think the "BlackOut" feature of [Lunar Pro](https://lunar.fyi/) by our good friend @alin23 is what you're looking for.

**Reply by waydabber** [reply]
I think BlackOut also disables the backlight and stops the amibent light sensor. But afaik Lunar can also read the ambient light sensor directly (something we did not implement in MonitorControl) which might still work even when the display is blacked out. @alin23 will probably chime in if this use-case is covered by Lunar Pro. :)

**Reply by DavidMali** [reply]
Thanks a lot for the quick replies!
My Problem with using Lunar was that their Blackout feature automatically mirrors a screen which broke my BetterDummy Setup. 
For now, I've managed to solve the issue with an App Store Application called "BlackOut", it cost me 2 Euros but it solves the issue. I can now use that app to black the internal monitor screen out while still preserving full MonitorControl and BetterDummy functionality. 

Thanks again, your apps are a blessing for someone using an external monitor. I'll be donating a bit to both projects to support development :)

**Reply by alin23** [reply]
Lunar now supports Blackout without mirroring and it can also use the MacBook sensor to adapt the external monitors while the builtin display is off. 

![image](https://user-images.githubusercontent.com/3819725/157425499-2cbe9a27-7c3e-402d-9182-a2121d276c29.jpeg)

Interesting that there's an App Store app called Blackout. Can you send a link to it?

**Reply by DavidMali** [reply]
Oh damn I didn't know about that, time to switch back to Lunar haha. Here's the link to the blackout app: https://apps.apple.com/de/app/blackout-display-manager/id1378187468?l=en&mt=12
